05/app.js

```diff
@@ -0,0 +1,60 @@
+document.addEventListener('DOMContentLoaded', init);
+
+function init() {
+    const form = document.querySelector('form');
+    if (form) {
+        form.addEventListener('submit', isValid)
+    };
+};
+
+function isValid(e) {
```

Comment:
Spróbuj może napisać bardziej elastycznie rozwiązanie, wzorując się na tym artykule:
https://devmentor.pl/b/walidacja-formularza-w-javascript#obs%C5%82uga-formularza-w-javascript-refaktoryzacja

To co zrobiłeś jest ok, ale to rozwiązanie byłoby mega ;)